[Org Settings] Update docs on new org settings (#28149)
DataDog/documentation: content/en/dashboards/sharing/shared_dashboards.md

Shared dashboards allow external viewers or users who prefer not to log into Datadog to access them. You can manage access using different sharing types, each with specific configuration options.

> All shared dashboards and their public accesss settings are listed in the [Public Sharing Settings page][3]. On the settings page, you can also add additional security configuration such as disabling specific sharing types, or setting maximum public access period.

Reviewer: ```suggestion
All shared dashboards and their public accesss settings are listed in the [Public Sharing Settings page][3]. You can also add additional security configurations, such as disabling specific sharing types, or setting the maximum public access period on this page.
```

> **Note**: When the shared dashboards feature is disabled, dashboards are no longer publicly accessible. However, their configurations remain viewable and editable.Disabling the feature does **not** affect the Active/Paused status of individual dashboards — even Active shared dashboards will not be publicly accessible.

Reviewer: ```suggestion
**Note**: When the shared dashboards feature is disabled, dashboards are no longer publicly accessible. However, their configurations remain viewable and editable. This feature operates independently of the `Active` or `Paused` status of individual dashboards, even active shared dashboards become publicly inaccessible.
```

Reviewer: Suggesting a minor clarification here, but let me know if you have any feedback

Reviewer 2: Reasonable change, thanks - suggesting that we update the last reference to 'active' to be capitalized and formatted the same as earlier instances @michlee1337

**Note**: Invited emails lose access at 12:00 a.m. local time on the expiration date.

> The maximum valid duration of an invitation can be configured by an organization admin in [**Organization Settings > Public Sharing**][3], it defaults to no maximum.

Reviewer: ```suggestion
The maximum valid duration of an invitation can be configured by an organization admin in [**Organization Settings > Public Sharing**][3]. By default, it is not configured.
```

> **Note**: Maximum valid duration is enforced by exact datetime. So if the org has a maximum valid duration of 1 day, an invite created at Jan 1st 11:00AM must expire before Jan 2nd 11:00AM.

Reviewer: ```suggestion
**Note**: Maximum valid duration is enforced by exact timestamp. For example, with a maximum valid duration of one day, an invitation created at Jan 1st 11:00AM expires before Jan 2nd 11:00AM.
```

Author: Small clarification here:
The invitation expiration is configured by the sharer.
So the org setting for "maximum valid duration" shows up as the last selectable expiration date for the sharer.

<img width="635" alt="image" src="https://github.com/user-attachments/assets/25d7a971-a285-4dd7-9d29-d42e729e700e" />

Reviewer: Thanks for that clarification!

## Embedded shared dashboards

You can embed shared dashboards into a website using an iframe. Access to these embedded dashboards is restricted to allowlisted request referrers.

> This feature is currently not supported on Safari web browsers.

Reviewer: ```suggestion
This feature is not supported on Safari web browsers.
```
